### PR TITLE
[YUNIKORN-1345] Updating the Spark tutorial i18n

### PR DESCRIPTION
**Pull request by wilfred-s**
fix includes correct assets link for i18n pages from 1.1.0 release

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md [hunk 1]
```diff
@@ -104,28 +108,39 @@ EOF
...
-export SPARK_HOME=/usr/local/spark-2.4.4-bin-hadoop2.7/
+export SPARK_HOME=/usr/local/spark/
 ${SPARK_HOME}/bin/spark-submit --master k8s://http://localhost:8001 --deploy-mode cluster --name spark-pi \
    --master k8s://http://localhost:8001 --deploy-mode cluster --name spark-pi \
    --class org.apache.spark.examples.SparkPi \
    --conf spark.executor.instances=1 \
    --conf spark.kubernetes.namespace=spark-test \
    --conf spark.kubernetes.executor.request.cores=1 \
-   --conf spark.kubernetes.container.image=apache/yunikorn:spark-2.4.4 \
+   --conf spark.kubernetes.container.image=docker.io/spark:v3.3.0 \
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md:121]: docker.io/spark:v3.3.0 should be docker.io/apache/spark:v3.3.0

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md [hunk 2]
```diff
@@ -104,28 +108,39 @@ EOF
...
+```
+:::
 
 您可以看见Spark的driver和executors在Kubernetes上创建:
 
-![spark-pods](./../../assets/spark-pods.png)
+![spark-pods](./../../assets/RunningSparkOnK8s.png)
 
-您还可以从 YuniKorn UI 查看作业信息。如果您不知道如何访问 YuniKorn UI，请阅读文档
+spark-pi 结果在驱动程序舱中。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md:140]: spark-pi结果在 driver pod中。

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md [hunk 3]
```diff
@@ -104,28 +108,39 @@ EOF
...
 [链接](../../get_started/get_started.md#访问-web-ui).
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md:141]: This png should be removed.

**Review by 0yukali0 — CHANGES_REQUESTED** [review]

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md (hunk 2 above)
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md:140]: updated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md (hunk 1 above)
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md:121]: updated

**Review by 0yukali0 — APPROVED** [review]
LGTM

**Pull request merged**